driver/src/main/scala/com/mongodb/scala/reactivestreams/client/MongoClient.scala

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.scala.reactivestreams.client
+
+import java.io.Closeable
+
+import com.mongodb.ConnectionString
+import com.mongodb.async.client.MongoClientOptions
+import com.mongodb.reactivestreams.client.{ MongoClient => JMongoClient, MongoClients }
+import org.bson.Document
+import org.reactivestreams.Publisher
+
+import scala.reflect.ClassTag
+
+object MongoClient {
```

Comment:
Its a companion object and the api the docs default to the case class - but have added some small doc text.